### DISCUSSION TITLE
Support Chains

### DISCUSSION THREAD

**Discussion by sheriff-rango** [rainbow-me/rainbowkit#1318] [Feedback]
Hi, I would like to know if `RainbowKit` supports cosmos based chains such as `Juno`, `Cosmos-Hub`, `Osmosis`, `Terra2`, etc.

**Comment by glitch-txs** [comment]
Only EVM chains

**Comment by DanielSinclair** [comment]
Currently only EVM chains are supported as this was historically what WalletConnect and Wagmi had supported. WalletConnect v2 has unlocked this a bit, but we would still require support in Wagmi. Something we'll watch!
